### PR TITLE
Added transform_point_to_circle

### PR DESCRIPTION
**Pull request by GeorgeEfstathiadis**
Added function to transform a center point to a shapely circus with a given radius.

### CODE REVIEW

**Comment by GeorgeEfstathiadis** [comment]
Sorry about that, but I am failing on `mypy` again. Do I add the modules in the `setup.py` and `requirements.txt` to pass the checks?

File: forest/jasmine/traj2stats.py [hunk 1]
```diff
@@ -1,14 +1,67 @@
...
+                                               write_all_summaries)
+from forest.jasmine.data2mobmat import (GPS2MobMat, InferMobMat,
+                                        great_circle_dist,
+                                        pairwise_great_circle_dist)
+from forest.jasmine.mobmat2traj import (Imp2traj, ImputeGPS, locate_home,
+                                        num_sig_places)
+from forest.jasmine.sogp_gps import BV_select
+
+
+def transform_point_to_circle(coords: Tuple[float, float], radius: int
```
Comment (hackdna) [inline forest/jasmine/traj2stats.py:30]: If `coords` has to be unpacked anyway then it makes sense to pass `lat` and `lon` directly instead.

File: forest/jasmine/traj2stats.py [hunk 2]
```diff
@@ -1,14 +1,67 @@
...
+    aeqd_to_wgs84 = partial(
+        pyproj.transform,
+        pyproj.Proj(local_azimuthal_projection),
+        pyproj.Proj("+proj=longlat +datum=WGS84 +no_defs"),
+    )
+
+    center = Point(*coords)
+    point_transformed = transform(wgs84_to_aeqd, center)
+    buffer = point_transformed.buffer(radius)
+    circle_poly = transform(aeqd_to_wgs84, buffer)
```
Comment (hackdna) [inline forest/jasmine/traj2stats.py:61]: No need for `circle_poly` variable, simply return here?

File: forest/jasmine/traj2stats.py [hunk 3]
```diff
@@ -1,14 +1,67 @@
+"""Module used to impute missing data, by combining functions defined in other
+modules and calculate summary statistics of imputed trajectories.
+"""
+
 import os
-import sys
 import pickle
+import sys
+import time
```
Comment (hackdna) [inline forest/jasmine/traj2stats.py:8]: Unused module?

File: forest/jasmine/tests/test_traj2stats.py [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    """
+
+    circle1 = transform_point_to_circle(coords1, 5)
+    point_in_edge = [
+        circle1.exterior.coords.xy[0][2], circle1.exterior.coords.xy[1][2]
+        ]
+
+    distance = great_circle_dist(
+        *coords1, *point_in_edge
+    )
```
Comment (hackdna) [inline forest/jasmine/tests/test_traj2stats.py:49]: This statement can fit on one line

**Review by hackdna — CHANGES_REQUESTED** [review]
Yes, you need to add Forest dependencies to `requires` list in `setup.py` (alphabetical order). Only dev dependencies go into `requirements.txt`.

**Comment by GeorgeEfstathiadis** [comment]
Still no luck with mypy failing.

File: forest/jasmine/traj2stats.py [hunk 5]
```diff
@@ -1,14 +1,63 @@
+"""Module used to impute missing data, by combining functions defined in other
+modules and calculate summary statistics of imputed trajectories.
+"""
+
 import os
-import sys
 import pickle
+import sys
+from typing import Tuple
```
Comment (hackdna) [inline forest/jasmine/traj2stats.py:8]: Unused import?

**Review by hackdna — CHANGES_REQUESTED** [review]
The mypy error message references several possible solutions: https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports

In this case the only way to solve this is to suppress this error for `shapely` because no type annotations for it exist. This is accomplished by adding the following section (in alphabetical order) to `mypy.ini`:
```
[mypy-shapely.*]
ignore_missing_imports = True
```

**Review by hackdna — APPROVED** [review]

**Pull request merged**